sites/geohub/src/components/util/MiniMap.svelte

```diff
@@ -37,8 +38,8 @@
 
 	export let metadata: RasterTileMetadata | VectorTileMetadata = undefined;
 	const is_raster: boolean = feature.properties.is_raster as unknown as boolean;
+	const product: string = feature.properties.tags?.find((tag) => tag.key === 'product')?.value;
```

Comment:
Could you move this logic into `RasterTileData` class if you don't use this variable in MiniMap component?